Engine/Fuego/Windows/KeyCodesWin.h

```diff
@@ -0,0 +1,241 @@
+#pragma once
+
+#include <Windows.h>
+
+#include "KeyCodes.h"
+
+static inline uint16_t GetKeyCode(int winKey)
+{
+    switch (winKey)
+    {
+        case VK_SPACE:
+            return 0;
```

Comment:
It should use the values from our enum, not magic constants
`return KeyCode::Space;`

Reply:
)))))))))))))))))))))))))))))))))))))))))